ADAL/src/ui/mac/ADCredentialCollectionController.m

```diff
@@ -36,11 +36,38 @@ -(id) init
     self = [super init];
     if(self)
     {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-        [NSBundle loadNibNamed:@"ADCredentialViewController" owner:self];
-#pragma clang diagnostic pop
-//        loaded = [[NSBundle mainBundle] loadNibNamed:@"ADCredentialViewController" owner:self topLevelObjects:nil];
+        //Generate the NTLM input dialog by code for Mac
+        //usename field
+        _usernameLabel = [[NSTextField alloc] initWithFrame:NSMakeRect(7, 36, 73, 17)];
+        SAFE_ARC_AUTORELEASE(_usernameLabel);
+        [_usernameLabel setStringValue:@"User Name"];
```

Comment:
Localization is done in ADNTLMUIPrompt.m, where those two labels' value will be set.
So the label values here are just placeholders, can be empty as well :)